src/config/common.py

```diff
@@ -394,3 +394,5 @@
         ]
     }
 }
+
+DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
```

Comment:
https://docs.djangoproject.com/en/3.2/releases/3.2/#customizing-type-of-auto-created-primary-keys